test/test_automl/test_estimators.py

```diff
@@ -698,8 +700,9 @@ def test_classification_pandas_support(self):
 
         # Make sure that at least better than random.
         # accuracy in sklearn needs valid data
-        y = automl._automl[0].InputValidator.validate_target(y)
-        self.assertTrue(accuracy(y, automl.predict(X)) > 0.5)
+        y = automl._automl[0].InputValidator.encode_target(y)
+        prediction = automl._automl[0].InputValidator.encode_target(automl.predict(X))
+        self.assertTrue(accuracy(y, prediction) > 0.5)
```

Comment:
It should be 0.555 as the dataset is unbalanced.

Reply:
thanks, I will do the change before the next push